CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ option(AV_BUILD_LAPACK "Enable building an embedded Lapack" ON)
 option(AV_BUILD_SUITESPARSE "Enable building an embedded SuiteSparse" ON)
 option(AV_BUILD_FFMPEG "Enable building an embedded FFMpeg" ON)
 option(AV_BUILD_ALICEVISION "Enable building of AliceVision" ON)
+option(AV_EIGEN_NO_ALIGN "Disable Eigen alignment" ON)
```

Comment:
@fabiencastan I used AV_EIGEN_NO_ALIGN because there's a variable with the same name in cctag library, so I thought that it makes sense to have similar naming in both projects.

Reply:
Oh good catch! We will have to change it in cctag to follow this rule.

Reply:
This is addressed in CCTag, resolving comment.